### PR TITLE
Update blocks.md

### PR DESCRIPTION
**Pull request by dev-kanishk**

### CODE REVIEW

File: docs/blocks.md [hunk 1]
```diff
@@ -237,7 +195,7 @@ new List.
 ---
 
 ### Parse JSON
-Stage: Experimental
+Stage: Removed
 
 > Warning: This block has been marked to removed in the future versions. This will be replace by a more generic parsing block.
```
Comment (saswatds) [inline docs/blocks.md:200]: Update this message to tell uses to use the send request block's config

File: docs/blocks.md [hunk 2]
```diff
@@ -20,24 +20,25 @@ The block is an living standard and blocks go through a life-cycle as mentioned
...
-[Group By](#Group-By) | Experimental |Groups an array based on a specified key 
-[Parse JSON](#Parse-JSON) | Experimental |Parses the incoming data to JSON | 21.09.1
+[Group By](#Group-By) | Experimental | Groups an array based on a specified key 
+[Parse JSON](#Parse-JSON) | Removed |Parses the incoming data to JSON | 21.09.1
 [Concatenate](#Concatenate) | Experimental | Take 2 data pieces and put them in a single array list
-[Condition](#Condition) | Experimental | Checks for the given condition and passes data to either "Accept" or "Reject" port
-[Check](#Check) | Experimental | Check for the given condition from 2 different inputs and pass data only if condition resolves to `True`
+[Condition](#Condition) | Experimental | Checks for the given condition and passes data to either "Accept" or "Reject" port | 21.11.1
+[Check](#Check) | Experimental | Check for the given condition from 2 different inputs and pass data only if condition resolves to `True` | 21.11.1
+[Create Durables](#Create-Durables) | Experimental | Set some value to be made durable | 21.12.2 | 21.12.2
```
Comment (saswatds) [inline docs/blocks.md:38]: Create data block is again missing!

File: docs/blocks.md [hunk 3]
```diff
@@ -20,24 +20,25 @@ The block is an living standard and blocks go through a life-cycle as mentioned
...
+[Assign Variables](#Assign-Variable)| Deprecated | Use Send Request's configuration to pass data to request variables |  | 21.12.2
```
Comment (saswatds) [inline docs/blocks.md:39]: What was updated in 21.12.2?

Comment (dev-kanishk) [reply docs/blocks.md:40]: Adding each and every change that we make to a block, will make things clumsy in this file.

File: docs/blocks.md (hunk 2 above)
Comment (dev-kanishk) [reply docs/blocks.md:38]: added

File: docs/blocks.md (hunk 3 above)
Comment (saswatds) [reply docs/blocks.md:40]: Where else is the source of truth?

**Review by saswatds — APPROVED** [review]

**Pull request merged**